main.cpp

```diff
@@ -59,6 +63,11 @@ void kv_store_global_api_example()
     res = kv_reset("/kv/");
     printf("kv_reset -> %d\n", err_code(res));
 
+    if (strcmp(STR(MBED_CONF_STORAGE_STORAGE_TYPE), "TDB_EXTERNAL") == 0) {
```

Comment:
In most projects, we normally do
```c
#if MBED_CONF_STORAGE_STORAGE_TYPE  = TDB_EXTERNAL
```
but kv-config doesn't define `TDB_EXTERNAL` as a macro, so we need to convert `MBED_CONF_STORAGE_STORAGE_TYPE` to string for string comparison (like everywhere `MBED_CONF_STORAGE_STORAGE_TYPE` is checked in Mbed OS)...